subsys/bluetooth/host/cs.c

```diff
@@ -41,4 +166,150 @@ int bt_cs_set_default_settings(struct bt_conn *conn,
 
 	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_SET_DEFAULT_SETTINGS, buf, NULL);
 }
+
+int bt_cs_read_remote_fae_table(struct bt_conn *conn)
+{
+	struct bt_hci_cp_le_read_remote_fae_table *cp;
+	struct net_buf *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_CS_READ_REMOTE_FAE_TABLE, sizeof(*cp));
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	cp = net_buf_add(buf, sizeof(*cp));
+	cp->handle = sys_cpu_to_le16(conn->handle);
+
+	return bt_hci_cmd_send_sync(BT_HCI_OP_LE_CS_READ_REMOTE_FAE_TABLE, buf, NULL);
+}
+
+void bt_hci_le_cs_read_remote_fae_table_complete(struct net_buf *buf)
+{
+	struct bt_conn *conn;
+	struct bt_conn_le_cs_fae_table fae_table;
+	struct bt_hci_evt_le_cs_read_remote_fae_table_complete *evt;
+
+	if (buf->len < sizeof(*evt)) {
+		LOG_ERR("Unexpected end of buffer");
+		return;
+	}
+
+	evt = net_buf_pull_mem(buf, sizeof(*evt));
+
+	if (evt->status) {
+		LOG_INF("Read Remote FAE Table failed with status 0x%02X", evt->status);
+		return;
+	}
+
+	conn = bt_conn_lookup_handle(sys_le16_to_cpu(evt->conn_handle), BT_CONN_TYPE_LE);
+
```

Comment:
minor: Zephyr Bluetooth stack style generally avoids empty lines between assignments to variables and testing the variable's value (same applies in any similar places in this PR)